### PR TITLE
[WFLY-8625] Create test case for Elytron Audit Logging

### PR DESCRIPTION
**Pull request by jtymel**
https://issues.jboss.org/browse/WFLY-8625
https://issues.jboss.org/browse/JBEAP-10488

### CODE REVIEW

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/audit/AbstractSyslogAuditLogTestCase.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    protected static boolean loggedSuccessfulAuth(BlockingQueue<SyslogServerEventIF> queue, String user) throws Exception {
+        return loggedAuthResult(queue, user, SUCCESSFUL_AUTH_EVENT);
+    }
+
+    protected static boolean loggedFailedAuth(BlockingQueue<SyslogServerEventIF> queue, String user) throws Exception {
+        return loggedAuthResult(queue, user, UNSUCCESSFUL_AUTH_EVENT);
+    }
+
+    protected static boolean loggedAuthResult(BlockingQueue<SyslogServerEventIF> queue, String user, String expectedEvent) throws Exception {
+        SyslogServerEventIF log = queue.poll(15L * 1000, TimeUnit.MILLISECONDS);
```
Comment (kwart) [inline testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/audit/AbstractSyslogAuditLogTestCase.java]: `(15L, TimeUnit.SECONDS)` ?

nitpicking :)

**Review by kwart — APPROVED** [review]
LGTM.

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/audit/AbstractSyslogAuditLogTestCase.java (hunk 1 above)
Comment (jtymel) [reply testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/audit/AbstractSyslogAuditLogTestCase.java]: damn you copy&paste
changed :)

**Review by kwart — APPROVED** [review]

**Review by kabir — APPROVED** [review]
Approving paperwork

**Pull request merged**